modules/ssh-agent-using.adoc

```diff
@@ -93,6 +93,11 @@ If you want to perform installation debugging or disaster recovery on your clust
 In a production environment, you require disaster recovery and debugging.
 ====
 
+[NOTE]
+====
+If your cluster is in FIPS mode, only use FIPS allowed algorithms to generate the ssh key. The key should be either rsa or ecdsa
```

Comment:
* ssh -> SSH
* rsa -> RSA
* ecdsa -> ECDSA
* Missing a period at the end.
* should -> must
* FIPS allowed -> FIPS-allowed?

Reply:
Maybe FIPS-compliant, that seems to be a common phrasing in Google search. Changes incorporated.